test/visual/ButtonVisual.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import ButtonToolbar from 'react-bootstrap/lib/ButtonToolbar';
 import Button from 'react-bootstrap/lib/Button';
-import { Link } from 'react-router';
+import { Link } from 'react-router-dom';
 
 import LinkContainer from '../../src/LinkContainer';
 
 export default () => (
   <div>
-    <Link to="home">Back to Index</Link>
+    <Link to="/home">Back to Index</Link>
```

Comment:
No, this is not necessary change - I updated URLs just for the sake of clarity as having absolute path looks more clear to me (you definitely know that it points to `/home` and not some `/something/home`).

Yes, sure, I'll definitely update changelog 👍